DEV-GUIDE.md

```diff
@@ -246,39 +251,29 @@ tooling functions correctly
 (You can safely ignore some errors like
 `com_github_opencontainers_go_digest Does not exist`)
 
-### Building the containers
+### Building and pushing the containers
 
-This tool builds and publishes Mixer container images to the specified
-registry.
+Build the containers in your local docker cache:
 
-```
-bin/publish-docker-images.sh -h gcr.io/my-project -t my-tag
+```shell
+sudo -E make docker
```

Comment:
should not need sudo

Reply:
I agree - sudo is not needed.  I had originally ran this without adding my id to the docker group.  As a result, docker.sock was not available.